apps/bfd-server/dev/api-changelog.md

```diff
@@ -1,5 +1,128 @@
 # API Changelog
 
+## BFD-2145: Removal of duplicated careteam member entries and their extensions.
+
```

Comment:
@williamharty can you assist with this section? Its hard to tell what the improvement is from the json blocks so I think some explanation in FHIR terms would be helpful.

Reply:
Corrected an issue where duplicate entries for a provider were being supplied in the Explanation of Benefit careTeam element, making it cumbersome to process and link care team members to items.